net/gajim/Portfile

```diff
@@ -24,40 +24,55 @@ supported_archs     noarch
 master_sites        https://dev.gajim.org/gajim/gajim/-/archive/${version}/
 use_bzip2           yes
 
-checksums           rmd160  050c844f9f2dc4682f2b6597b3ef4ad642e9c416 \
-                    sha256  50d1f5f5376e52c2fb1d481b790877f19ed987d68ea9f39c915dd16286b6837d \
-                    size    3076987
+checksums           rmd160  44756e3aea4e6a918c8b9ed63a38b69f330e80f1 \
+                    sha256  c6064bece82aeb603e7e93102bebbf28a779f08a68ba7067ad56f821f0faea13 \
+                    size    3112338
 
-variant python310 conflicts python311 description {Use Python 3.10} {}
-variant python311 conflicts python310 description {Use Python 3.11} {}
+variant python310 conflicts python311 python312 description {Use Python 3.10} {}
+variant python311 conflicts python310 python312 description {Use Python 3.11} {}
+variant python312 conflicts python310 python311 description {Use Python 3.12} {}
 
-if {![variant_isset python310]} {
-    default_variants +python311
+if {![variant_isset python310] && ![variant_isset python311]} {
+    default_variants +python312
 }
 
-foreach pv {311 310} {
+foreach pv {312 311 310} {
     if {[variant_isset python${pv}]} {
         python.default_version ${pv}
         break
     }
 }
 
-depends_build       port:gettext \
+depends_build-append \
+                    port:gettext \
                     port:intltool \
-                    path:bin/pkg-config:pkgconfig \
-                    port:libtool
+                    port:libtool \
+                    path:bin/pkg-config:pkgconfig
 
-depends_lib         port:adwaita-icon-theme \
+depends_lib-append  port:adwaita-icon-theme \
                     path:lib/pkgconfig/gtk+-3.0.pc:gtk3 \
                     port:gtksourceview4 \
                     path:lib/pkgconfig/librsvg-2.0.pc:librsvg \
                     path:lib/pkgconfig/libsoup-2.4.pc:libsoup \
                     port:py${python.version}-certifi \
                     port:py${python.version}-css-parser \
+                    port:py${python.version}-emoji \
                     port:py${python.version}-idna \
                     port:py${python.version}-keyring \
                     port:py${python.version}-nbxmpp \
                     port:py${python.version}-omemo-dr \
                     port:py${python.version}-pillow \
```

Comment:
not your change but this should be  "py-Pillow" to avoid breakage on case-sensitive file systems.